Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -385,8 +385,7 @@ open class OptimizelyClient: NSObject {
                                      featureEnabled: false)
             return false
```

Comment:
We still need to send an impression event with a dummy metadata even when pair is null. These lines should be refactored to support it.

Reply:
We also need to change the impression-event from activate call. When the experimentKey is for feature-test (not ab-test) we should send a valid flag-key (instead of copying experiment-key).